src/components/calendar/CalendarPage.jsx

```diff
@@ -0,0 +1,320 @@
+import PeopleIcon from '@material-ui/icons/People';
+import PersonIcon from '@material-ui/icons/Person';
+import {
+  Avatar,
+  Button,
+  ButtonGroup,
+  Chip,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  FormControl,
+  Grid,
+  Input,
+  InputAdornment,
+  InputLabel,
+  Paper,
+  Select,
+  Tab,
+  Tabs,
+  TextField,
+} from '@material-ui/core';
+
+const styles = theme => ({
+  title: {
+    fontDuration: 'normal',
+    marginLeft: '30px',
+  },
+  flex: {
+    display: 'flex',
+    [theme.breakpoints.down('sm')]: {
+      display: 'block',
+    },
+  },
+  calendar: {
+    // margin: '0 10px',
+    height: '700px',
+  },
+  component: {
+    width: 'calc(100% / 7)',
+    color: '#fff',
+    height: 'fit-content',
+    textAlign: 'center',
+    [theme.breakpoints.down('sm')]: {
+      width: 'inherit',
+    },
+  },
+  label: {
+    padding: '7.5px 0',
+    backgroundColor: '#280e3a',
+    margin: '0',
+  },
+  bar: {
+    width: '5px',
+    height: '84px',
+    backgroundColor: '#e91e63',
+    borderRadius: '15px',
+  },
+  detail: {
+    color: '#000',
+    fontSize: '12px',
+    textAlign: 'left',
+    lineHeight: '1',
+    marginLeft: '10px',
+  },
+  event: {
+    display: 'flex',
+  },
+  header: {
+    justifyContent: 'space-between',
+  },
+  button: {
+    border: 'none',
+    cursor: 'pointer',
+    display: 'flex',
+    fontSize: '16px',
+    outline: 'none',
+    [theme.breakpoints.down('sm')]: {
+      margin: 'auto',
+    },
+  },
+  icon: {
+    margin: 'auto',
+  },
+  iconLabel: {
+    marginLeft: '10px',
+    marginRight: '30px',
+  },
+  duration: {
+    padding: '5px 30px',
+    borderColor: '#765ea8',
+  }
+});
+
+class CalendarPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false,
+      background: ['#280e3a', '#fff', '#fff', '#fff'],
+      color: ['#fff', '#000', '#000', '#000']
+    };
+  }
+
+  handleOpen() {
+    this.setState({ open: true });
+  }
+
+  handleClose() {
+    this.setState({ open: false });
+  }
+
+  handleDelete() {
+
+  }
+
+  handleChange() {
+
+  }
+
+  handleSelect(selected) {
+    let background = this.state.background;
+    let color = this.state.color;
+
+    for (let i = 0; i < 4; i++)
+      if (i === selected) {
+        background[i] = '#280e3a';
+        color[i] = '#fff';
+      }
+      else {
+        background[i] = '#fff';
+        color[i] = '#280e3a';
+      }
+
+    this.setState({ background: background });
+  }
+
+  render() {
+    const { classes } = this.props;
+    return <div>
+      <div className={clsx(classes.header, classes.flex)}>
+        <h1 className={classes.title}>Calendar</h1>
+        <Button className={classes.button} onClick={this.handleOpen.bind(this)}>
+          <EventIcon className={classes.icon}></EventIcon>
+          <p>Schedule Interview</p>
+        </Button>
+      </div>
+      <Paper style={{ margin: '17px 10px auto' }} square>
```

Comment:
I try to keep all the styling in the style const at the top of the file. It helps with debugging the styles if we need to make changes later on :)